inf-clojure.el

```diff
@@ -725,19 +731,24 @@ The value is nil if it can't find one."
 (defun inf-clojure-show-var-documentation (var)
   "Send a form to the inferior Clojure to give documentation for VAR.
 See function `inf-clojure-var-doc-form'."
-  (interactive (inf-clojure-symprompt "Var doc" (inf-clojure-var-at-pt)))
-  (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-doc-form) var)))
+  (interactive "P")
+  (let ((var (if (inf-clojure--should-prompt-on-commands var)
```

Comment:
Can you expand on this? It is working fine here, prompting when the function returns true.